### ISSUE TITLE
Investigate batch normalization

### ISSUE THREAD

**Issue by vgangal101** [vgangal101/resnet_models#7]
Check that your batch norm works as intended

Check 
https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization

Check 
https://stackoverflow.com/questions/47538391/keras-batchnormalization-axis-clarification
